### DISCUSSION TITLE
GitHub asks me to put my real name publicly???

### DISCUSSION THREAD

**Discussion by ghost** [community/community#60778] [GitHub Education] [answered]
So, I applied the second time, and I get 
```
You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio. The verification process looks at your GitHub profile to cross reference the evidence submitted. Do you use a variation of your name or a nickname? Is your name in non-english? When you update your profile information please log out and log back into GitHub before re-applying.
```
Do I have to literally put my real name? My profile it's linked from other platforms and those verifications take like 10 days each, I'm afraid bad actors could do something bad with it, is there an alternative? I can't make my profile private temporally either, so I guess I'm gonna have to give up on this.

**Comment by the-officialjosh** [comment, marked as answer]
That is their only way of checking if the name on your ID is the owner of the GitHub ACC

**Reply by notscar** [reply]
How would that prove anything? I've logged in with my 2FA code to submit the request, and if i was to use somone else documents i would have no problem to set my name temporarily to his

**Reply by the-officialjosh** [reply]
I don't know why it was done that way honestly but like in my younger brother's case we just changed it immediately after he was accepted and nothing happened we were still in the program

**Reply by ghost** [reply]
I'm guessing this is my only alternative left, the only problem being that in all my attempts they took like 4-7 days which is kinda a lot for me but I guess I'm gonna have to 😞.

**Comment by radar-blip** [comment]
According to the answer here: #52888, you can change it back again after its been verified.
